FeatureAreas.props

```diff
@@ -221,7 +221,10 @@
   <!-- Dependencies for ImageIcon -->
   <PropertyGroup Condition="Exists('InnerLoopAreas.props') And $(SolutionName) == 'MUXControlsInnerLoop' And $(FeatureImageIconEnabled) == 'true'">
   </PropertyGroup>
-    <!-- Dependencies for AnimatedIcon -->
+  <!-- Dependencies for ColorFilterOverlayControl -->
+  <PropertyGroup Condition="Exists('InnerLoopAreas.props') And $(SolutionName) == 'MUXControlsInnerLoop' And $(FeatureColorFilterOverlayControlEnabled) == 'true'">
```

Comment:
I think that the inner loop feature areas need to be updated as well.

Reply:
Can you be more specific about what is missing or what needs to change?

Reply:
You will need to add this feature to the DatePicker  group in line 73 since we need to build this project if want the styles for DatePicker to work as expected.

Reply:
Thats right, DatePicker should not be dependent on ColorFilterOverlayControl so this [PropertyGroup](https://github.com/microsoft/microsoft-ui-xaml/blob/4289186bb8c803627e0b785c0ad58ce335f4f389/FeatureAreas.props#L73) needs to add that as a `ProductOnly` dependency.